scripts/shared/nodeinfo/nodeinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+    "location": {
+        "continent": "NA",
+        "country": "Canada",
+        "region": "Ontario",
+        "municipality": "Toronto",
+        "uri": "https://tomesh.net"
+    },
+    "software": {
```

Comment:
@makeworld-the-better-one It's a really odd way to do this with a partial json file that isn't actually valid json. It's better to have a properly terminated valid json here, use macros like `__REPLACE_ME__` if needed, then use `jq` to add additional elements.

Reply:
Ok, will do.

Reply:
Done as well, I've used sed to replace stuff though, if that's ok.